oak_restricted_kernel_sdk/src/dice.rs

```diff
@@ -127,23 +200,51 @@ impl RecipientContextGenerator for EncryptionKeyHandle {
     }
 }
 
-pub struct Attester {
+pub struct InstanceAttester {
```

Comment:
Attester cannot be Instance vs Group, because it doesn't depend on the Key Provisioning. So it should just be Attester

Reply:
Disagree. I think it's helpful to state that this attester retrieves the instance evidence, even if we never implement a `GroupAttester` (which would rely key provisioning. 

The other `instance` structs do not depend on key provisioning either.

Reply:
I think the existence of an InstanceAttester implies that there will be a GroupAttester

Reply:
And maybe there never will be. Even so, imo it avoids confusion by making it clear which type of evidence is exposed.